crates/hir-ty/src/mir/eval.rs

```diff
@@ -272,6 +272,7 @@ const STACK_OFFSET: usize = 1 << 30;
 const HEAP_OFFSET: usize = 1 << 29;
 
 impl Address {
+    #[allow(clippy::double_parens)]
     fn from_bytes(it: &[u8]) -> Result<Self> {
         Ok(Address::from_usize(from_bytes!(usize, it)))
     }
```

Comment:
Feels like this lint should not trigger on macro expansions 🤔

Reply:
Good point, I will report it and the other bug as well